src/test/java/com/wikia/webdriver/testcases/adstests/TestAdTypeMobile.java

```diff
@@ -27,6 +27,7 @@ public void adsAdTypeAsyncSuccessWithAd() {
     // https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=260851332&orderId=245575332
     Page page = new Page("project43", "SyntheticTests/AdType/Async/Success");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)